### PR TITLE
Added AdminAbsoluteUrlMixin

### PR DESCRIPTION
**Pull request by ar4s**
Added new mixin (`AdminAbsoluteUrlMixin`) for automatically generate URL
based on model.

### CODE REVIEW

File: src/ralph/assets/models/mixins.py [hunk 1]
```diff
@@ -64,3 +60,11 @@ def save(self, update_last_seen=False, *args, **kwargs):
         if update_last_seen:
             self.last_seen = timezone.now()
         super(LastSeenMixin, self).save(*args, **kwargs)
+
+
+class AdminAbsoluteUrlMixin(object):
+    def get_absolute_url(self):
+        info = self._meta.app_label, self._meta.model_name
+        return reverse(
+            'admin:{}_{}_change'.format(*info), args=(self.pk,)
```
Comment (ghost) [inline src/ralph/assets/models/mixins.py]: could it be simply:
`'admin:{}_{}_change'.format(self._meta.app_label, self._meta.model_name), args=(self.pk,)`
? ;)

File: src/ralph/assets/models/mixins.py [hunk 2]
```diff
@@ -1,9 +1,5 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
-
+from django.core.urlresolvers import reverse
 from django.db import models
 from django.utils import timezone
 from django.utils.encoding import python_2_unicode_compatible
```
Comment (szok) [inline src/ralph/assets/models/mixins.py]: Delete this line

File: src/ralph/assets/models/mixins.py [hunk 3]
```diff
@@ -64,3 +60,12 @@ def save(self, update_last_seen=False, *args, **kwargs):
         if update_last_seen:
             self.last_seen = timezone.now()
         super(LastSeenMixin, self).save(*args, **kwargs)
+
+
+class AdminAbsoluteUrlMixin(object):
+    def get_absolute_url(self):
```
Comment (mkurek) [inline src/ralph/assets/models/mixins.py:63]: maybe some unit test? :smirk:

Comment (szok) [inline src/ralph/assets/models/mixins.py:63]: :+1:

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**